common/production_methods/11_private_infrastructure.txt

```diff
@@ -0,0 +1,298 @@
+		}
+	}
+
+	state_modifiers = {
+		workforce_scaled = {
+			state_infrastructure_add = 20
+		}
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			market_land_trade_capacity_add = 1
+		}
+	}
+}
+
+pm_electric_trains = {
+	texture = "gfx/interface/icons/production_method_icons/trains_electric.dds"
+	pollution_generation = 10
+	
+	unlocking_technologies = {
+		electric_railway
+	}
+
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_engines_add = 15
+			building_input_electricity_add = 15
+			building_output_transportation_add = 75
+		}
+		level_scaled = {
+			building_employment_laborers_add = 2000
+			building_employment_machinists_add = 2000
+			building_employment_clerks_add = 750
+		}
+	}
+
+	state_modifiers = {
+		workforce_scaled = {
+			state_infrastructure_add = 30
+		}
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			market_land_trade_capacity_add = 1.5
+		}
+	}
+}
+
+pm_diesel_trains = {
+	texture = "gfx/interface/icons/production_method_icons/trains_diesel.dds"
+	pollution_generation = 10
+	
+	unlocking_technologies = {
+		compression_ignition
+	}
+
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_engines_add = 20
+			building_input_oil_add = 20
+			building_output_transportation_add = 100
+		}
+		level_scaled = {
+			building_employment_laborers_add = 1500
+			building_employment_machinists_add = 2500
+			building_employment_clerks_add = 750
+		}
+	}
+
+	state_modifiers = {
+		workforce_scaled = {
+			state_infrastructure_add = 40
+		}
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			market_land_trade_capacity_add = 2
+		}
+	}
+}
+
+pm_no_passenger_trains = {
+	texture = "gfx/interface/icons/production_method_icons/no_passenger_trains.dds"
+}
+
+pm_wooden_passenger_carriages = {
+	texture = "gfx/interface/icons/production_method_icons/passenger_trains.dds"
+	pollution_generation = 10
+
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_wood_add = 5
+			building_output_transportation_add = 15
+		}
+		level_scaled = {
+			building_employment_machinists_add = -250
+			building_employment_clerks_add = -250
+		}
+	}
+	
+	state_modifiers = {
+		workforce_scaled = {
+			state_infrastructure_add = -5
+		}
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			market_land_trade_capacity_add = -0.5
+		}
+	}				
+}
+
+pm_steel_passenger_carriages = {
+	texture = "gfx/interface/icons/production_method_icons/passenger_carriages.dds"
+	pollution_generation = 10
+
+	unlocking_technologies = {
+		steel_railway_cars
+	}
+
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_steel_add = 5
+			building_output_transportation_add = 30
+		}
+		level_scaled = {
+			building_employment_machinists_add = -500
+			building_employment_clerks_add = -500
+		}
+	}
+	
+	state_modifiers = {
+		workforce_scaled = {
+			state_infrastructure_add = -5
+		}
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			market_land_trade_capacity_add = -1
```

Comment:
these land_trade_capacity need to be divide by 10. At least according to my sense of balance.